src/Ploi/Resources/NetworkRule.php

```diff
@@ -40,7 +40,7 @@ public function get(int $id = null): Response
         return $this->getPloi()->makeAPICall($this->getEndpoint());
     }
 
-    public function create(string $name, int $port, string $type = 'tcp', string $fromIpAddress): Response
+    public function create(string $name, int $port, string $type = 'tcp', string $ruleType = 'allow', string $fromIpAddress): Response
```

Comment:
Please add this parameter after `$fromIpAddress`, as it is now it is a breaking change because of people using this SDK and using the fromIpAddress wil now convert to the ruleType which isn't valid.